kore/src/Kore/Repl/Data.hs

```diff
@@ -200,6 +210,74 @@ data GraphView
     | Expanded
     deriving (Eq, Show)
 
+-- | Log options which can be changed by the log command.
+data GeneralLogOptions =
+    GeneralLogOptions
+        { logType :: !Log.KoreLogType
+        , logLevel :: !Log.Severity
+        , timestampsSwitch :: !Log.TimestampsSwitch
+        , logEntries :: !Log.EntryTypes
+        }
+    deriving (Eq, Show)
+
+generalLogOptionsTransformer
+    :: GeneralLogOptions
+    -> Log.KoreLogOptions
+    -> Log.KoreLogOptions
+generalLogOptionsTransformer
+    logOptions@(GeneralLogOptions _ _ _ _)
```

Comment:
Do you need to pattern match here?

Reply:
Yes, when a new field is added to `GeneralLogOptions` this will not type check anymore. This way we won't forget to modify this function as well.